### PR TITLE
Build(deps): Bump sidekiq from 6.4.1 to 6.4.2

### PR DESCRIPTION
**Pull request by tgxworld**

### CODE REVIEW

File: spec/lib/backup_restore/system_interface_spec.rb [hunk 1]
```diff
@@ -107,12 +107,13 @@
...
     context "with Sidekiq workers" do
-      before { flush_sidekiq_redis_namespace }
-      after { flush_sidekiq_redis_namespace }
+      after do
+        flush_sidekiq_redis_namespace
+      end
 
       def flush_sidekiq_redis_namespace
         Sidekiq.redis do |redis|
-          redis.scan_each { |key| Discourse.redis.del(key) }
```
Comment (tgxworld) [inline spec/lib/backup_restore/system_interface_spec.rb:115]: This was causing a leak in our test suite as we were deleting from the wrong namespace.

File: spec/lib/backup_restore/system_interface_spec.rb [hunk 2]
```diff
@@ -144,7 +145,7 @@ def create_workers(site_id: nil, all_sites: false)
             run_at: Time.now.to_i,
             payload: Sidekiq.dump_json(payload)
           )
-          conn.hmset("#{key}:workers", '444', data)
+          conn.hmset("#{key}:work", '444', data)
```
Comment (tgxworld) [inline spec/lib/backup_restore/system_interface_spec.rb:148]: This got renamed too...

**Review by CvX — APPROVED** [review]
btw. why was CLAassistant triggered? wrong email in the commit?

**Comment by tgxworld** [comment]
Initially I pushed amended dependabot's commit to include my changes xD

**Pull request merged**